hive/src/main/java/com/netease/arctic/hive/CachedHiveClientPool.java

```diff
@@ -73,7 +73,7 @@ private synchronized void init() {
   @Override
   public <R> R run(Action<R, HMSClient, TException> action) throws TException, InterruptedException {
     try {
-      return tableMetaStore.doAs(() -> clientPool().run(action));
+      return tableMetaStore.doAs(() -> clientPool().run(action, true));
```

Comment:
What problem is this going to fix? Is there another PR to fix this?

Reply:
I have also committed this change in another pr and I will delete it in this pr.